### PR TITLE
Added `@global` documentation in several files

### LINKED ISSUE #601: @global documentation is missing in several files
`@global` documentation is missing in below files.

- includes/Admin/Admin_AJAX.php
- includes/Checker/Abstract_Check_Runner.php
- includes/Checker/Runtime_Environment_Setup.php
- includes/Utilities/Plugin_Request_Utility.php

### PR DESCRIPTION
**Pull request by shail-mehta**
Fixes: https://github.com/WordPress/plugin-check/issues/601

### CODE REVIEW

**Review by ernilambar — APPROVED** [review]

File: includes/Admin/Admin_AJAX.php [hunk 1]
```diff
@@ -145,6 +145,9 @@ public function set_up_environment() {
 	 * Handles the AJAX request to cleanup the runtime environment.
 	 *
 	 * @since 1.0.0
+	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 * @global string $table_prefix The database table prefix.
```
Comment (mukeshpanchal27) [inline includes/Admin/Admin_AJAX.php]:
```suggestion
	 * @global wpdb   $wpdb         WordPress database abstraction object.
	 * @global string $table_prefix The database table prefix.
```

File: includes/Checker/Abstract_Check_Runner.php [hunk 2]
```diff
@@ -319,6 +319,9 @@ final public function prepare() {
 	 *
 	 * @since 1.0.0
 	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 * @global string $table_prefix The database table prefix.
```
Comment (mukeshpanchal27) [inline includes/Checker/Abstract_Check_Runner.php]:
```suggestion
	 * @global wpdb   $wpdb         WordPress database abstraction object.
	 * @global string $table_prefix The database table prefix.
```

File: includes/Checker/Runtime_Environment_Setup.php [hunk 3]
```diff
@@ -18,6 +18,10 @@ final class Runtime_Environment_Setup {
 	 * Sets up the WordPress environment for runtime checks
 	 *
 	 * @since 1.0.0
+	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 * @global string $table_prefix The database table prefix.
+	 * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
```
Comment (mukeshpanchal27) [inline includes/Checker/Runtime_Environment_Setup.php]:
```suggestion
	 * @global wpdb               $wpdb          WordPress database abstraction object.
	 * @global string             $table_prefix  The database table prefix.
	 * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
```

**Review by mukeshpanchal27 — CHANGES_REQUESTED** [review]

**Comment by shail-mehta** [comment]
Thank you for your feedback. I have made the requested changes.

**Review by mukeshpanchal27 — APPROVED** [review]

**Pull request merged**